charts/bookkeeper/templates/bookkeeper.yaml

```diff
@@ -14,6 +14,7 @@ spec:
   zookeeperUri: {{ .Values.zookeeperUri }}
   envVars: {{ template "bookkeeper.fullname" . }}-configmap
   autoRecovery: {{ .Values.autoRecovery }}
+  blockOwnerDeletion: {{ .Values.blockOwnerDeletion }}
```

Comment:
we can set this field to false only when blockownerDeletion is set to false in values.yaml file.

Reply:
A similar check could be added to provide a default value for this newly added field
```
{{ default true .Values.blockOwnerDeletion }}
```

Reply:
Made the changes